hedera/templates/cms/home_page.html

```diff
@@ -11,7 +11,7 @@
 
 {% block body_base %}
 
-  <section class="home-hero">
+  <section class="home-hero" title="Hedera Logo">
```

Comment:
I'd maybe title it "Hedera Header" instead as it's really the whole header section, not just the logo.

Reply:
done.